application/src/main/data/json/system/widget_types/gateway_configuration.json

```diff
@@ -8,7 +8,15 @@
     "type": "static",
     "sizeX": 8,
     "sizeY": 6.5,
-    "resources": [],
+    "resources": [
+      {
+        "url": {
+          "entityType": "TB_RESOURCE",
+          "id": "${RESOURCE:gateway-management-extension.js}"
```

Comment:
typically our resource is identified as an
'/api/images/system/scada_fluid_system_bundle_image.png' or /api/images/public/1ZtrwcvVuuNPdFQ3hsqA4wL0U4bIJuEL

Reply:
This will be done as a separate feature on UI. The API is ready